tests/build.proj

```diff
@@ -28,32 +46,19 @@
   <Target Name="BatchRestorePackages" DependsOnTargets="VerifyDependencies">
     <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Restoring all packages..." />
     
-    <!-- restore all project.jsons in one pass for perf & to avoid concurrency problems -->
-    <Exec Command="$(DnuRestoreCommand) $(DnuRestoreDirs)"
-          StandardOutputImportance="Low" />
-          
+    <!-- restore all csproj's with PackageReferences in one pass -->
+    <MSBuild Projects="build.proj"
+             Properties="RestoreProj=%(RestoreProjects.Identity)"
+             Targets="RestorePackage" />
+
     <Message Importance="High" Text="[$([System.DateTime]::Now.ToString('HH:mm:ss.ff'))] Restoring all packages...Done." />
   </Target>
 
-  <!-- Task from buildtools that validates dependencies contained in project.json files. -->
-  <UsingTask TaskName="ValidateProjectDependencyVersions" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" />
-
-  <!-- Tasks from buildtools for easy project.json dependency updates -->
-  <UsingTask TaskName="UpdatePackageDependencyVersion" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" />
-
-  <!-- Create a collection of all project.json files for dependency updates. -->
-  <ItemGroup>
-    <ProjectJsonFiles Include="$(SourceDir)**\project.json" />
-    <ProjectJsonFiles Include="$(ProjectDir)scripts\**\project.json" />
-  </ItemGroup>
-
-  <Target Name="UpdatePackageDependencyVersion">
-    <UpdatePackageDependencyVersion ProjectJsons="@(ProjectJsonFiles)"
-                                    PackageId="$(PackageId)"
-                                    OldVersion="$(OldVersion)"
-                                    NewVersion="$(NewVersion)" />
+  <Target Name="RestorePackage">
+    <Exec Command="$(DnuRestoreCommand) $(RestoreProj)"
```

Comment:
DotnetRestoreCommand in general you we should not have any references to Dnu anywhere any longer.